astroid/inference_tip.py

```diff
@@ -6,20 +6,19 @@
 
 from __future__ import annotations
 
-import sys
 from collections.abc import Callable, Iterator
+from typing import TYPE_CHECKING
 
 from astroid.context import InferenceContext
 from astroid.exceptions import InferenceOverwriteError, UseInferenceDefault
 from astroid.nodes import NodeNG
-from astroid.typing import InferenceResult, InferFn
-
-if sys.version_info >= (3, 11):
-    from typing import ParamSpec
-else:
-    from typing_extensions import ParamSpec
-
-_P = ParamSpec("_P")
+from astroid.typing import (
+    _P,
```

Comment:
Is there a better way to do this/name this?

Reply:
I think in general people tend to just redefine a `ParamSpec` per file, can't we do that here?